docs/graphs/classdiagram_lib.dot

```diff
@@ -0,0 +1,34 @@
+digraph hierarchy {
+size="5,5"
+node[   shape=record,
+        style=filled,
+        fillcolor=gray95]
+edge[dir=back, arrowtail=none]
+
+
+ps[label = "{ProgramState|
+                all_transactions\l
+                filtered_transactions\l|
+                + setup()\l
```

Comment:
I need documentation on what `setup`, `add`, `apply`, `categorize` and `store` will do. Is it possible to add some short documentation to the class?

Reply:
I would suggest a separate file with the class doc, that we can later convert to do strings. I would remove `apply(filter)` because I think now, it's better done by the frontend.